gulpfile.js

```diff
@@ -20,7 +20,7 @@ var yargs = require('yargs');
 var path = require('path');
 var fs = require('fs');
 var htmllint = require('gulp-htmllint');
-var package = require('./package.json');
+var _package = require('./package.json');
```

Comment:
it seems odd to me to have one variable with an underscore while none of the others have one. what do you think about calling it `pkg` instead of `_package`?